services/src/performance-tests/src/main/groovy/org/openkilda/performancetests/helpers/TopologyHelper.groovy

```diff
@@ -45,20 +45,18 @@ class TopologyHelper extends org.openkilda.functionaltests.helpers.TopologyHelpe
             def region = i % regions.size()
             topo.addCasualSwitch("${managementControllers[region]} ${statControllers[region]}") 
         }
-        islsAmount.times {
+        //create links between neighbor switches
+        def allSwitches = topo.getSwitches()
+        for (def i = 0; i < allSwitches.size() - 1; i++) {
+            topo.addIsl(allSwitches[i], allSwitches[i+1])
+        }
+        //create links between random switches
+        islsAmount-topo.getIsls().size().times {
```

Comment:
I have doubts about this working as you expect. Round braces missing? 
`(islsAmount-topo.getIsls().size()).times {`
Also, this code may throw because you can end up with negative number here `islsAmount-topo.getIsls().size()`

Reply:
yep, round braces were missed.
L54 should not fail in case negative number(it should just to return `null`)